src/System.Reflection.Metadata.Cil/SignatureDecoder.cs

```diff
@@ -44,16 +44,16 @@ internal static ImmutableArray<CilType> DecodeMethodSpecificationSignature(BlobH
             return NewDecoder(provider).DecodeMethodSpecificationSignature(ref blobReader);
         }
 
-        internal static CilType DecodeType(EntityHandle handle, CilTypeProvider provider, bool? isValueType)
+        internal static CilType DecodeType(EntityHandle handle, CilTypeProvider provider, byte? isValueType)
```

Comment:
This parameter name looks wrong now. Should be rawTypeCode, I believe.

Reply:
Yes, that is correct. What does that rawTypeCode means? 0 if it is class and 1 if it is valuetype?